docs/resources/google_compute_instance.md.erb

```diff
@@ -75,11 +75,17 @@ The following examples show how to use this InSpec audit resource.
       its('first_network_interface_type'){ should eq "one_to_one_nat" }
     end
 
+### Test that a particular compute instance label key is present
+
+    describe google_compute_instance(project: 'chef-inspec-gcp',  zone: 'us-east1-b', name: 'inspec-test-vm') do
+      its('labels_keys') { should include 'my_favourite_label' }
+    end
+
 <br>
 
 ## Properties
 
-*  `cpu_platform`, `creation_timestamp`, `deletion_protection`, `disks`, `id`, `kind`, `label_fingerprint`, `machine_type`, `metadata`, `name`, `network_interfaces`, `scheduling`, `start_restricted`, `status`, `tags`, `zone`
+*  `cpu_platform`, `creation_timestamp`, `deletion_protection`, `disks`, `id`, `kind`, `label_fingerprint`, `machine_type`, `metadata`, `name`, `network_interfaces`, `scheduling`, `start_restricted`, `status`, `tags`, `zone`, `labels_keys`, `labels_values`
```

Comment:
We should talk about a tagging / label API.  Azure does it with dynamic methods; AWS doesn't really do it at all yet.  We should do it in a similar way, and I think a custom matcher might make more sense than a pair of properties.

Reply:
that would be great - let's start with what's there and iterate

Reply:
I'd be reluctant to add anything you want to remove later.  It's very hard to deprecate once thing are in the wild.  Since this isn't in core, let's ship it, but keep in mind if we have doubts about something, better to not expose it.

Reply:
See https://github.com/inspec/inspec/issues/3139